dev-test/badstock.py

```
@@ -0,0 +1,76 @@
+    print("3. Sell Stock")
+    print("4. View Portfolio")
+    print("5. Exit")
+
+def view_stocks():
+    for stock, price in stocks.items():
+        # Prices fluctuate wildly each time you look
+        new_price = price + random.randint(-100, 100)
+        stocks[stock] = new_price
+        print(f"{stock}: ${new_price}")
+
+def buy_stock():
```

Ruff (0.8.2)

Missing return type annotation for public function `buy_stock`

Add return type annotation: `None`

(ANN201)